api/user.json

```diff
@@ -154,32 +148,30 @@
     "climateActionIncentiveIsRural": null
   },
   "refund": {
-    "Total Payable": {
```

Comment:
IS changing this going to break something somewhere? It doesn't seem used, but I really don't know :)

Reply:
nope! I added it just for this page for now